app/views/gds_design_system_main_template.scala.html

```diff
@@ -0,0 +1,82 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+
+@this(
+        govukHeader: GovukHeader,
+        govukLayout: GovukLayout,
+        govukPhaseBanner: GovukPhaseBanner,
+        govukBackLink: GovukBackLink,
+        siteHeader: components.siteHeader,
+        phaseBanner: components.phaseBanner
+)
+
+
+@(  pageTitle: Option[String] = None,
+    backButtonTitle: String = "Back",
+    backButtonUrl: String = "/")(contentBlock: Html)(implicit request: Request[_], messages: Messages)
+
+@head = {
+    <link rel="shortcut icon" href="/lib/govuk-frontend/govuk/assets/images/favicon.ico" type="image/x-icon" />
```

Comment:
Assets controller ?

Reply:
I am already bringing this, but nothing changes.

Reply:
Dummy!  Thanks! :)